p2p/src/simulated/network.rs

```diff
@@ -349,19 +348,6 @@ impl<E: RNetwork<Listener, Sink, Stream> + Spawner + Rng + Clock> Network<E> {
                         return;
                     }
 
-                    // Drop message if too large
-                    if message.len() > max_size {
-                        trace!(
-                            recipient = hex(&recipient),
-                            channel,
-                            size = message.len(),
-                            max_size,
-                            reason = "message too large",
-                            "dropping message",
-                        );
-                        return;
-                    }
-
```

Comment:
removed for being unreachable code, I could change it to an assert or something though

Reply:
I think the new behavior of the link "dying" (as the recv_frame loop will stop) matches how it works in p2p::authenticated.